packratt/cache.py

```diff
@@ -90,6 +90,9 @@ def __setitem__(self, key, value):
         with open(entry_dir / "entry.yaml", "w") as f:
             f.write(yaml.safe_dump(value))
 
+    def set_cache_entry(self, key, entry):
```

Comment:
Do we need this? Can we not just use 

```
cache[key] = value
```

which invokes ``Cache.__setitem__``?